entity-management-web/src/main/java/eu/europeana/entitymanagement/batch/processor/EntityDereferenceProcessor.java

```diff
@@ -25,11 +33,13 @@
     private static final Logger logger = LogManager.getLogger(EntityDereferenceProcessor.class);
     private final MetisDereferenceService dereferenceService;
     private final EntityComparator entityComparator;
+    private final ValidatorFactory emValidatorFactory;
 
     @Autowired
-    public EntityDereferenceProcessor(MetisDereferenceService dereferenceService) {
+    public EntityDereferenceProcessor(MetisDereferenceService dereferenceService, ValidatorFactory emValidatorFactory) {
```

Comment:
the minimal validation must not be integrated into the dereferencing step, but as own functionality in the EntityUpdateProcessor see step 4 in the specs https://docs.google.com/document/d/16k9PcCMFwl2LXjnnzotZRPc-QqM-Ar1D0VELHt4t_hA/edit#

Reply:
Improved.